File: 2021/1.py
with open("1.in", "r") as fh:
    lines = fh.readlines()
    prev = 0
    lc = 0
    inc = 0
    for line in lines:
        if int(line) > prev and lc > 0:
            inc += 1
        lc += 1
        prev = int(line)
    print(inc)

File: 2021/1.php
<?php
$r = file_get_contents("1.in");
$l = explode("\n", $r);
$inc = 0;
$prev = 0;
foreach ($l as $i => $v) {
    if (($v > $prev) && $i) {
        $inc++;
    }
    $prev = $v;
}
echo $inc;
